### ISSUE TITLE
Mutations

### ISSUE THREAD

**Issue by the-vampiire** [the-vampiire/hacktoberithms#3]
Credit: [adapted from Free Code Camp](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-algorithm-scripting/mutations/)

# Challenge
Write a function that satisfies the following rules:
- Return true if the string in the first element of the list contains all of the letters of the string in the second element of the list.

## examples
`["hello", "Hello"]`
- should return true because all of the letters in the second string are present in the first, ignoring case.

`["hello", "hey"]`
- should return false because the string "hello" does not contain a "y".

`["Alien", "line"]`
- should return true because all of the letters in "line" are present in "Alien".

**Comment by yoch** [comment]
It's worth to precise that letter count matter.

> ['hel', 'heel'] # should be False

**Comment by mikelane** [comment]
Agreed, the directions do not make it clear that repeated letters must be accounted for.
